kubernetes/client/clientset/clientset.go

```diff
@@ -1,96 +1,23 @@
 package clientset
 
 import (
-	composev1beta1 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta1"
-	composev1beta2 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta2"
-	"github.com/golang/glog"
-	"k8s.io/client-go/discovery"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/util/flowcontrol"
+	api "github.com/docker/compose-on-kubernetes/api/client/clientset"
 )
 
 // Interface defines the methods a compose kube client should have
 // FIXME(vdemeester) is it required ?
-type Interface interface {
-	Discovery() discovery.DiscoveryInterface
-	ComposeV1beta2() composev1beta2.ComposeV1beta2Interface
-	ComposeV1beta1() composev1beta1.ComposeV1beta1Interface
-}
+type Interface = api.Interface
```

Comment:
To encourage people to move to the new package, might want to officially mark them deprecated by adding a deprecation comment.

Not sure what the best description would be to clearly direct people to the right location (@vdemeester any suggestions?). Perhaps something like;

```suggestion
// Deprecated: Use github.com/docker/compose-on-kubernetes/api/client/clientset.Interface instead
type Interface = api.Interface
```

(Same for all the other aliases we created)

Reply:
@thaJeztah yes that works, most of the IDE/editor should pick those (and mark them as deprecated in some way)